### DISCUSSION TITLE
dependencies attribute doesn't work inside repeatable field

### DISCUSSION THREAD

**Discussion by rickyviz** [Laravel-Backpack/community-forum#265] [Q&A (Help)]
dependencies attribute doesn't work inside repeatable field
i using  a select2 field that depends on another field, but cant get dependencies, using dd`($form = backpack_form_input());` but repeatable request still null, need this for using in query parameter 

![image](https://user-images.githubusercontent.com/89818695/150165510-37361927-994b-4c97-b742-8f4101d9d2c0.png)![image](https://user-images.githubusercontent.com/89818695/150165329-5889745d-c1e6-44d3-9ab0-ce2a8e98390c.png)

**Comment by rickyviz** [comment]
Up

**Reply by pxpm** [reply]
Hey @rickyviz , hope to find you well. Thanks for the bump 👍 😅 

I had this on radar, but I was delaying the answer until I had something to help you with. It took more time than expected, for that I apologise, I already had alot of conversations about this feature, for example here https://github.com/Laravel-Backpack/CRUD/pull/3439 

I think now I have something that I can give you that are not just words: https://github.com/Laravel-Backpack/CRUD/pull/4211

This PR should fix the issue parsing the repeatable values. Please read in the PR description my field setup, it seems you are using `include_all_form_fields` too many times. 

If that PR does not fix it for you, it would help me if you can provide me your Backpack setup, you can get it running the following command: `php artisan backpack:version`. 

Thanks and sorry once again 👍
Pedro

**Reply by rickyviz** [reply]
sorry i cant get it , field **sow_id** on **repeatable**i using to make params to vendor_id
![image](https://user-images.githubusercontent.com/89818695/160844909-b6a63612-3192-4391-ae08-eee040981dcb.png)
still null 
![image](https://user-images.githubusercontent.com/89818695/160845356-4b62f601-bb16-45c2-ac98-07bbfe6cf37d.png)

**Reply by pxpm** [reply]
@rickyviz sorry man, but I can't help you more if you don't help me here with more information. Can you please give me the output of the command: `php artisan backpack:version` ? You need to be running atleas `5.0.9` to have the fix, and making sure you don't have the `repeatable` field overriden in your `resources` folder, otherwise Backpack will use your version and not the updated one.

Cheers

**Comment by jcastroa87** [comment]
This will be close due to not activity, feel free to reopen or create a new one if needed.

Cheers.
